portfolio/src/main/webapp/calendar.js

```diff
@@ -69,14 +103,70 @@ function getTimeObject(timeString) {
   return new Date(currentYear, currentMonth, currentDate, timeHour, timeMinute);
 }
 
-/** Creates a single element for a new calendar event. */
+/** Creates a card element for a new calendar event. */
+/** 
+ * TODO(hollyyuqizheng) Look into refactoring the card creation code 
+ * into a JavaScript class. This card element will be used in different 
+ * sections on the UI, so this refactoring will make things simpler. 
+ */
 function updateCalendarEventList(newCalendarEvent) {
```

Comment:
Let's just have one method comment rather than two. Creates [...] and TODO can belong to the same method comment.

/**
 * Creates...
 * TODO(...): ...

Reply:
Updated. I actually deleted the comment because I think the refactoring might not be too efficient. Tasks and calendar events have different fields and therefore different card structure, so I think leaving the creation of cards in each object's own file is not too bad of a choice.

Reply:
Updated.